react/ComponentDocs.tsx

```diff
@@ -1,15 +1,13 @@
-import React, { Fragment, FunctionComponent } from 'react'
+import React, { FunctionComponent } from 'react'
 import { Query, compose, graphql } from 'react-apollo'
 import { branch, renderComponent, renderNothing } from 'recompose'
 import { ApolloError } from 'apollo-client'
-import { Helmet, withRuntimeContext, NoSSR } from 'vtex.render-runtime'
+import { withRuntimeContext } from 'vtex.render-runtime'
```

Comment:
unnecessary

Reply:
I actually need to use `withRuntimeContext` so that I can have access to the `runtime` down in the bottom of the file, when I'm using `compose` with `graphql()` to make another GraphQL query.
This was hidden by GitHub view hahaha